azurerm/internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -130,6 +137,7 @@ func resourceArmLogAnalyticsWorkspaceCreateUpdate(d *schema.ResourceData, meta i
 	}
 
 	retentionInDays := int32(d.Get("retention_in_days").(int))
+	dailyQuotaGb := float64(d.Get("daily_quota_gb").(float64))
```

Comment:
I don't think you need the cast here as you're asserting a `float64`, rather than converting between types?